frb_rust/src/into_into_dart.rs

```diff
@@ -0,0 +1,172 @@
+use crate::{ffi::*, DartSafe};
+
+/// Basically the Into trait.
+/// We need this separate trait because we need to implement it for Vec<T> etc.
+/// These blanket implementations allow us to accept external types in various places.
+/// The initial reason for this was to allow mirrored types in StreamSink<>.
+/// See also [PR 1285](https://github.com/fzyzcjy/flutter_rust_bridge/pull/1285)
+pub trait IntoIntoDart<D> {
```

Comment:
nit: should we constrain that `D: IntoDart`? because the name `IntoIntoDart` does imply this

Reply:
Good idea, I added that.